server/Helpers/signIn.js

```diff
@@ -3,6 +3,10 @@ import jwt from "jsonwebtoken";
 // todo: reduce this time and implement refresh token
 const ACCES_TOKEN_EXPIRATION = 86400; // 24h in secondes
 
```

Comment:
To be really cool I'd create an `accesTokenOption` object and an `accestTokenName` varriable, like that:
```typescript
const accestTokenName = "accesToken";
const accesTokenOption = {
  httpOnly: true
  // todo: add `secure: true` to only send token in https
  // secure: true,
};
```

And use it like that:
```typescript
// in deleteAccessTokenCookie
export const deleteAccessTokenCookie = (res) => {
  res.clearCookie(accestTokenName, accesTokenOption);
};
```
```typescript
// in setAccesTokenCookie
res.cookie(accestTokenName, accesToken, {
    ...accesTokenOption,
    maxAge: ACCES_TOKEN_EXPIRATION * 1000
  });
```

With that we can change the token name, add/remove option really easily